src/messages/index.ts

```diff
@@ -21,4 +23,9 @@ export const messages = {
   enabled: 'enabled',
   disabled: 'disabled',
   restart: '\'Restart\' for changes to take effect.',
+  ng2Detected: 'VSCode-Icons has detected an Angular 2 project. ' +
```

Comment:
To be precise, Angular 4 in March, and then a new version every six months according to what the team has been publishing lately